package.json

```
@@ -57,7 +57,7 @@
     "debug": "4.4.0",
     "execa": "4.1.0",
     "istanbul-lib-coverage": "^3.0.0",
-    "js-yaml": "4.1.0",
+    "js-yaml": "4.1.1",
```

Maybe we could also make this more future-proof? Not sure why we have `^` versioning for some but not all dependencies.

```suggestion
    "js-yaml": "^4.1.1",
```